src/main/resources/bootstrap.yaml

```diff
@@ -6,3 +6,6 @@ spring:
       paths: /mnt/secrets/reform-scan
       aliases:
         app-insights-instrumentation-key: azure.application-insights.instrumentation-key
+        storage-account-name: storage.account_name
+        storage-account-primary-key: storage.account_key
+        storage-account-secondary-key: storage.account_secondary_key
```

Comment:
should those be like env vars used in application.yaml?

Reply:
yeah, done